packages/framework/attributor/src/mixinAttributor.ts

```diff
@@ -112,8 +112,46 @@ export const mixinAttributor = (Base: typeof ContainerRuntime = ContainerRuntime
 			runtimeOptions: IContainerRuntimeOptions | undefined = {},
 			containerScope: FluidObject | undefined = context.scope,
 			existing?: boolean | undefined,
-			ctor: typeof ContainerRuntime = ContainerRuntimeWithAttributor as unknown as typeof ContainerRuntime,
+			containerRuntimeCtor: typeof ContainerRuntime = ContainerRuntimeWithAttributor as unknown as typeof ContainerRuntime,
 		): Promise<ContainerRuntime> {
```

Comment:
Should we deprecate this `public static async load` function as well?  Feels weird to keep it around when it won't work (due to `provideEntryPoint` throwing)

Reply:
This isn't technically exported/visible. This mixin pattern on static methods is very weird to me